integration/servicenow-petportal-demo/README.md

```diff
@@ -0,0 +1,6 @@
+# ServiceNow Pet Portal Demo
+
+This is my Pet Portal includes extra governance in Service Now.  This blueprint also leverages the xlr-xlrelease-plugin to dynamically tag the releases with the Service Now tickets.  To setup this showcase you should use the included [docker-compose.yml](docker-compose.yml) file.  The *xld-xlrelease-plugin* will need to be stored relative to the *docker-compose.yml* at *xlr/plugins/xlr-xlrelease-plugin-2.3.4.jar*.
```

Comment:
What is the value of this blueprint for users that want to instantiate it? What will it get them?

Reply:
Many potential customers and new customers need and want some guidance on how various tools can be orchestrated with XL Release.  I have done a number of demos for potential customers who are very impressed with the idea that release could orchestrated automatic and manual tasks as well as multiple ticketing systems.  It is also helpful to show ways that SNOW could be used in a release process.  This template is based on many conversations.

Reply:
I understand that users want to see something like that in a demo.

But why should this be a blueprint? A blueprint is meant to help users get started with their own project. How many users do you expect to have a project called "PetPortal" that requires integration with ServiceNow?

In any case, my question was simpler. This `README.md` does not describe why a user would want to use this blueprint.